doc/nrf/releases_and_maturity/releases/release-notes-1.9.0.rst

```diff
@@ -174,20 +174,20 @@ nRF9160: Asset Tracker v2
 
   * Support for A-GPS filtered ephemerides.
   * Functionality that allows the application to wait for A-GPS data to be processed before starting GNSS positioning.
-  * New documentation for the following modules in :ref:`asset_tracker_v2`:
+  * New documentation for the following modules in Asset Tracker v2:
 
-    * :ref:`asset_tracker_v2_app_module`
-    * :ref:`asset_tracker_v2_cloud_module`
-    * :ref:`api_cloud_wrapper`
-    * :ref:`asset_tracker_v2_data_module`
+    * Application module
+    * Cloud module
+    * Cloud wrapper API
+    * Data module
     * GNSS module
-    * :ref:`asset_tracker_v2_modem_module`
-    * :ref:`api_modules_common`
-    * :ref:`asset_tracker_v2_sensor_module`
-    * :ref:`asset_tracker_v2_ui_module`
-    * :ref:`asset_tracker_v2_util_module`
+    * Modem module
+    * Modules common
+    * Sensor module
+    * UI module
+    * Utility module
 
-  * Support for atmospheric pressure readings retrieved from the BME680 sensor on Thingy:91. See :ref:`asset_tracker_v2_sensor_module` for more information.
+  * Support for atmospheric pressure readings retrieved from the BME680 sensor on Thingy:91. See sensor module for more information.
```

Comment:
```suggestion
  * Support for atmospheric pressure readings retrieved from the BME680 sensor on Thingy:91. See Asset Tracker v2 sensor module for more information.
```

Reply:
Why keep all these references to Asset Tracker v2? This sounds confusing given this PR is meant to remove it. Does this mean this is not a full removal? I suggest a migration guide entry, as there is none at the moment.

Reply:
These are in the old release notes. We normally keep as plain text and just remove the ref link. I think it is fine. 

Wouldn't a change log entry be enough, instead of a migration guide entry? We normally don't add removal of a lib or application in the migration guide is my understanding (Correct me if I am wrong). For example, Zigbee removal.